### DISCUSSION TITLE
updated , but error with cannot import name 'RootModel' from 'pydantic'

### DISCUSSION THREAD

**Discussion by joelhoty** [lllyasviel/stable-diffusion-webui-forge#1279] [General]
"I’ve updated the program, and after that, an error occurred as shown below. How can I solve this?"

ImportError: cannot import name 'RootModel' from 'pydantic' (D:\Flux\system\python\lib\site-packages\pydantic\__init__.cp310-win_amd64.pyd)

**Comment by kerizane** [comment]
Probably fixed by the latest commit.
